core/src/main/java/org/testcontainers/containers/DockerComposeContainer.java

```diff
@@ -395,6 +396,10 @@ default void validateFileList(List<File> composeFiles) {
  * Use Docker Compose container.
  */
 class ContainerisedDockerCompose extends GenericContainer<ContainerisedDockerCompose> implements DockerCompose {
+
+    private static final String DOCKER_SOCKET_PATH = "//var/run/docker.sock";
```

Comment:
should not start with double `/` (you add it in `getDockerSocketHostPath`)

Reply:
You are right, still works in Linux interestingly ^^

Reply:
well, on *nix systems, you can use as many `/` as you want :D
```bash
$ ls -la /////usr
total 0
drwxr-xr-x@  10 root  wheel    320 Sep 27 08:04 .
drwxr-xr-x   31 root  wheel    992 Nov 30 13:02 ..
drwxr-xr-x  976 root  wheel  31232 Nov 30 13:02 bin
drwxr-xr-x  263 root  wheel   8416 Nov 16 11:38 include
drwxr-xr-x  311 root  wheel   9952 Nov 30 13:02 lib
drwxr-xr-x  235 root  wheel   7520 Nov 30 13:02 libexec
drwxr-xr-x   15 root  wheel    480 Sep 26 22:58 local
drwxr-xr-x  246 root  wheel   7872 Nov 30 12:51 sbin
drwxr-xr-x   47 root  wheel   1504 Sep 27 08:04 share
drwxr-xr-x    5 root  wheel    160 Sep 21 06:32 standalone
```